docs/src/index.md

```diff
@@ -17,12 +17,12 @@ It is for example possible to directly use a `DataFrame` from [*DataFrames.jl*](
 Pkg.add("PGFPlotsX")
 ```
 
-*PGFPlots.jl* requires a latex installation with the PGFPlots package installed.
-We recommend using a latex installation with access to lualatex since it can have significantly better performance than pdflatex.
+*PGFPlots.jl* requires a LaTeX installation with the PGFPlots package installed.
+We recommend using a LaTeX installation with access to `lualatex` since it can have significantly better performance than `pdflatex`.
 
-To generate or preview figures in svg (like is done by default in Jupyter notebooks) `pdf2svg` is required. This can obtained by, on Ubuntu, running `sudo apt-get install pdf2svg`, on RHEL/Fedora `sudo dnf install pdf2svg` and on macOS e.g. `brew install pdf2svg`. On Windows, the binary can be downloaded from [here](http://www.cityinthesky.co.uk/opensource/pdf2svg/); be sure to add `pdf2svg` to the `PATH`.
+To generate or preview figures in `svg` (like is done by default in Jupyter notebooks) `pdf2svg` is required. This can obtained by, on Ubuntu, running `sudo apt-get install pdf2svg`, on RHEL/Fedora `sudo dnf install pdf2svg` and on macOS e.g. `brew install pdf2svg`. On Windows, the binary can be downloaded from [here](http://www.cityinthesky.co.uk/opensource/pdf2svg/); be sure to add `pdf2svg` to the `PATH`.
```

Comment:
Maybe `` [`svg`](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) ``

Reply:
I ended up not doing this, because I consider this a well-known acronym given the context. Same applies to `pdf` etc.